.github/workflows/pre-release.yml

```diff
@@ -10,30 +10,49 @@ jobs:
     name: "Pre-release"
     runs-on: "ubuntu-latest"
     container: ghcr.io/opencs-ontology/ci-worker:main
-
+    
     steps:
       - name: Checkout repository
         uses: actions/checkout@v2
         with:
           path: opencs
 
       - name: "Package ontology"
-        run: python /app/package.py opencs package dev
-
+        run: python /app/package.py opencs package dev;
+          
+      - name: "Download robot.jar"
+        uses: wei/wget@v1
+        with:
+          args: https://github.com/ontodev/robot/releases/download/v1.9.1/robot.jar
+        
+      - name: "Java installation"
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin' 
+          java-version: '17'   
+          
+      - name: "Infer additional assertions"
+        run: |
+          bash opencs/inference/infer_assertions.sh
+          touch inferred_assertions.ttl
+          bash opencs/inference/ofn_to_ttl.sh inferred_assertions.ofn inferred_assertions.ttl
+          rm output_opencs.ttl opencs2.ttl
+             
       - uses: "marvinpinto/action-automatic-releases@v1.2.1"
         with:
           repo_token: "${{ secrets.GITHUB_TOKEN }}"
           automatic_release_tag: "dev"
           prerelease: true
           title: "Development build"
           files: package/*
-
+      
       - name: "Prepare ontology files for commit"
         run: |
-          mkdir output_files
+          mkdir output_files 
           gzip -cd package/opencs.ttl.gz > output_files/opencs.ttl
           gzip -cd package/opencs.rdf.gz > output_files/opencs.rdf
           gzip -cd package/opencs.nt.gz > output_files/opencs.nt
+          mv inferred_assertions.ttl output_files/inferred_assertions.ttl
```

Comment:
Actually, this part is responsible for pushing ontology files into Github Pages repository, not for pushing to the releases.

Reply:
We will fix that, thanks for comment on this and clarification.

Reply:
If you could actually elaborate on which indentations should be removed. We would appreciate it.

Reply:
Lines 13, 48 and 51.

Reply:
thank you